i18n/en.json

```diff
@@ -2832,10 +2832,10 @@
   "navbar_dropdown.keyboardShortcuts": "Keyboard Shortcuts",
   "navbar_dropdown.leave": "Leave Team",
   "navbar_dropdown.leave.icon": "Leave Team Icon",
-  "navbar_dropdown.logout": "Logout",
+  "navbar_dropdown.logout": "Log Out",
   "navbar_dropdown.manageGroups": "Manage Groups",
   "navbar_dropdown.manageMembers": "Manage Members",
-  "navbar_dropdown.marketplace": "Plugin Marketplace",
+  "navbar_dropdown.marketplace": "Plug-in Marketplace",
```

Comment:
Question: "Plugin" is used throughout the docs, website etc. Would this change require those instances to also be changed?

Reply:
Yes, it would.